pkg/cli/acorn.go

```diff
@@ -120,21 +120,21 @@ func (a *Acorn) PersistentPre(cmd *cobra.Command, args []string) error {
 			return err
 		}
 	}
-
-	if a.Project != "" {
-		clientFactory := CommandClientFactory{
-			cmd:   cmd,
-			acorn: a,
-		}
-		cfg, err := config.ReadCLIConfig()
-		if err != nil {
-			return err
-		}
-
-		err = project.Exists(cmd.Context(), clientFactory.Options().WithCLIConfig(cfg), a.Project)
-		if err != nil {
-			return err
-		}
+	cfg, err := config.ReadCLIConfig()
+	if err != nil {
+		return err
+	}
+	clientFactory := CommandClientFactory{
+		cmd:   cmd,
+		acorn: a,
+	}
+	//check default project to ensure it exists in the current context
+	if a.Project == "" {
+		a.Project = cfg.CurrentProject
+	}
```

Comment:
Seems like this PR is adding tolerance for a.Project == "". When would this happen?

Reply:
so a.Project is only set when -j/--project is used in a cmd 

from the disscusion in standup @iwilltry42 was running into the issue where if you have a project default set and swap kubeconfigs you will not be alerted that the previously set default does not exist in that new context

this change will not check if no project is specified in the -j/--project flag for a cmd to check the currently set default project to make sure it exists

Reply:
Awesome, and then it would hit that error condition. Makes sense! Seems like a test could be written in a github workflow most easily, but would be funky to have a single test written in there. Maybe a new job inside of test.yml for this?